src/components/Home/Home.js

```diff
@@ -88,6 +92,7 @@ function Home(props) {
                   color: textColors[i]?.trim(),
                 }}
                 alt={altTexts[i]?.trim()}
+                icon={icons[i]?.trim()}
```

Comment:
Could this be

```javascript
                icon={icons && icons[i]?.trim()}
```

That way you don't have to have the if statement and null check above and then assign to `[]`?  Not a big deal, just a recommendation.